cmd/plume/types.go

```diff
@@ -92,11 +92,24 @@ type ReleaseMetadata struct {
 }
 
 type BuildMetadata struct {
-	CommitHash string `json:"commit"`
-	Version    string `json:"version"`
-	Endpoint   string `json:"endpoint"`
+	CommitHash []Commit `json:"commits"`
+	Version    string   `json:"version"`
+	Endpoint   string   `json:"endpoint"`
 }
 
 type Metadata struct {
 	LastModified string `json:"last-modified"`
 }
+
+type IndividualReleaseMetadata struct {
```

Comment:
I think at some later point (even in another PR) it would make sense to split structures belonging to different JSON-documents in different source files. It took me a bit to realize where this one was coming from.

Reply:
I feel like that might cause confusion though if the FCOS metadata types are defined in separate files while every other struct used in plume is inside of this file though. Is this maybe more of an issue with the individual names of these structs rather than the location?

Reply:
I'm fine either ways (and also keeping as-is). It was just a minor sentiment when scanning through the source.